### PR TITLE
Task 39 class Handler and test.

### PR DESCRIPTION
**Pull request by pauriarte**

### CODE REVIEW

File: lib/Handler.pm [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+use strict;
+use warnings;
+use AddUserCmd;
+use DelUserCmd;
+use PurchaseCmd;
+use PostaPkgCmd;
+use DispatchCmd;
+use ReceptionCmd;
+#use StateOrderCmd;
```
Comment (manuellerena) [inline lib/Handler.pm]: Remove commented line

**Comment by manuellerena** [comment]
+1

**Comment by earandap** [comment]
+1

File: lib/Handler.pm [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+};
+
+sub command_factory {
+    my $self = shift;
+    my $line = shift;
+    my @parameters = $self->$parse_line($line);
+    my $command;
+    
+     
+    use v5.18;
```
Comment (elin3t) [inline lib/Handler.pm:59]: @pauriarte porque este 'use ' en esta linea? esta un poco raro

File: t/handler.t [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+#my $command = $handler->command_factory("Y");
+#is($command->isa('StateOrderCmd'),1,"command_factory stateordercmd");
+
+
+
+
+done_testing();
+
+
```
Comment (elin3t) [inline t/handler.t:96]: Is a little bit scary to see this warnings in the test:

``` shell
perl -t handler.t 
given is experimental at ../lib/Handler.pm line 60.
when is experimental at ../lib/Handler.pm line 61.
when is experimental at ../lib/Handler.pm line 64.
when is experimental at ../lib/Handler.pm line 67.
when is experimental at ../lib/Handler.pm line 70.
when is experimental at ../lib/Handler.pm line 73.
when is experimental at ../lib/Handler.pm line 76.
when is experimental at ../lib/Handler.pm line 79.
when is experimental at ../lib/Handler.pm line 82.
when is experimental at ../lib/Handler.pm line 85.
ok 1 - command_factory
ok 2 - run_command
ok 3 - print_errors
1..3
```

**Pull request merged**